packages/back-end/src/services/auth/index.ts

```diff
@@ -248,6 +254,45 @@ export async function processJWT(
         dateCreated: new Date(),
       });
     };
+
+    if (IS_CLOUD) {
+      const build = getBuild();
+      req.gb = new GrowthBook({
+        apiHost: "https://cdn.growthbook.io",
+        clientKey: GB_SDK_ID,
+        enableDevMode: true,
+        stickyBucketService: new BrowserCookieStickyBucketService({
+          jsCookie: req.cookies,
+        }),
+        trackingCallback: () => {
+          // TODO: How to send event to Jitsu?
```

Comment:
@jdorn Is there a way to use Jitsu here? Or do we have an alternative for back-end logging?